src/ucp/core/ucp_context.c

```diff
@@ -1443,6 +1474,8 @@ static ucs_status_t ucp_fill_config(ucp_context_h context,
 
 err_free:
     ucs_free(context->config.alloc_methods);
+err_strdup:
```

Comment:
maybe better - it would be more clear what kind of rollback action we will do:
```
err_free_alloc_methods:
    ucs_free(context->config.alloc_methods);
err_free_env_prefix:
    ucs_free(context->config.env_prefix);
err:
    UCP_THREAD_LOCK_FINALIZE(&context->mt_lock);
    return status;
```